.github/workflows/ci.yaml

```diff
@@ -3,20 +3,24 @@ name: Build
 on:
   pull_request:
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
-  cancel-in-progress: true
-
 jobs:
-  ci:
+  lint:
     runs-on: ubuntu-latest
-    name: CI 👷
+    name: Lint 👷
     steps:
       - name: Check out source code
-        uses: actions/checkout@main
+        uses: actions/checkout@v4
 
       - name: Setup OpenTofu
-        uses: opentofu/setup-opentofu@main
-    
-      - name: Initialize Tofu project
+        uses: opentofu/setup-opentofu@v1
+
+      - name: OpenTofu fmt
+        run: tofu fmt -recursive -diff -check
+        continue-on-error: true
```

Comment:
whats the reason for this line? 👀

Reply:
If the fmt fails, we can just report the error and continue to also run `tofu validate`.

Reply:
That avoid a situation where it fail early because of wrong fmt, you fix, push, and it fail again because of `tofu validate`